tools/topology/platform/intel/dmic.m4

```diff
@@ -46,13 +46,15 @@ define(`DMIC_CONFIG',
 `		SOF_TKN_INTEL_DMIC_DUTY_MAX'		STR($5)
 `		SOF_TKN_INTEL_DMIC_SAMPLE_RATE'		STR($6)
 `		SOF_TKN_INTEL_DMIC_FIFO_WORD_LENGTH'	STR($7)
+`		SOF_TKN_INTEL_DMIC_UNMUTE_RAMP_TIME_MS'	STR($8)
```

Comment:
Is there anything Intel specific about DMIC ramp time. It's probably be needed by other DMICs too so could be a generic token.

Reply:
@lgirdwood Yep this is generic for any system but only if we keep this as time. (@plbossart was considering to convert this to the gain coefficient. Such would be totally SOF FW driver code specific thing.)

My own favorite is keep this e.g. as "SOF_TKN_DMIC_UNMUTE_RAMP_TIME_MS" and compute value in kernel (since we want to move it out from FW).